package.json

```diff
@@ -37,12 +37,8 @@
     "lint:sass": "yarn lint:sass:nofix --fix",
     "test:js": "jest",
     "test:js:watch": "jest --watchAll",
-    "test:js:coverage": "JEST_JUNIT_OUTPUT_DIR='./coverage' jest --ci --runInBand --reporters=default --reporters=jest-junit",
-    "test:js:report-coverage": "cat ./coverage/lcov.info | coveralls",
-    "test:js:check-coverage": "nyc check-coverage --temp-dir coverage",
```

Comment:
I think we could also remove `coveralls` and `nyc` deps entirely.

Reply:
Fixed in b5467deddab77a622a98c8033ef2d27decf87e2e